layers/+lang/typescript/README.org

```diff
@@ -27,12 +27,15 @@ This layer provides:
 - Imenu
 - linting
 - tsx mode
+- formatting
 
 * Install
 ** Pre-requisites
 You will need =node.js v0.12.0= or greater
 
-If you want linting run:  =npm instell -g typescript=  =npm install -g tslint=  
+If you want linting run:  =npm install -g typescript=  =npm install -g tslint=  
```

Comment:
Better be sure. Besides I took it from here https://www.npmjs.com/package/tslint  to be sure
Than I changed `install` to `instell` and moved `-g` :smile: